deploy/complete/terraform/mushop.tf

```diff
@@ -17,9 +17,30 @@ resource "helm_release" "mushop" {
   namespace = kubernetes_namespace.mushop_namespace.id
   wait      = false
 
-  values = [
-    file("${path.module}/../helm-chart/mushop/values-mock.yaml"),
-  ]
+  set_string {
+    name  = "global.mock.service"
+    value = var.mushop_mock_mode_all ? "all" : "false"
+  }
+  set {
```

Comment:
I don't think these conditional settings are necessary. If mock mode is `"all"` then these values are essentially ignored

Reply:
You mean this one: ```name  = "global.oadbAdminSecret"
    value = var.mushop_mock_mode_all ? "" : var.db_admin_name```

Reply:
?

Reply:
the three `global.oadb*`

Reply:
removed unnecessary fear conditions.